core/identity-management/iin-agent/README.md

```diff
@@ -33,3 +33,43 @@ make build
 ## Running the IIN Agent
 
 Run `npm run dev` to start the agent in your terminal. (_Note_: we will support a containerized IIN agent in the future.)
+
+## Configuration
+
+### Security Domain Config
+
+Sample `security-domain-config.json`:
+```
+{
+    "securityDomainName": "<ledgerId>",
+    "securityDomainName": [
+        "<node1/org1>",
+        "<node2/org2>"
+    ]
+}
+```
+
+It is JSON, with specifying who are the members of a security domain, with keys being the security domain.
```

Comment:
The above structure is a bit misleading. You should put `securityDomainName` in angle brackets (`<>`) to indicate that it needs to be replaced by a name. Also, you should have just one key-value entry in the above example, with the possibilities for the value being listed below rather than in the structure.

Reply:
It could be an array of strings (for Corda, its array of nodes) or just a string (for Fabric, its channel name), so I'm not sure how to explain both in one key-value entry?